src/deprecated/src/conditionals/the-events-calendar-conditional.php

```diff
@@ -4,15 +4,20 @@
 
 /**
  * Conditional that is only met when The Events Calendar exists.
+ *
+ * @deprecated 19.12
  */
 class The_Events_Calendar_Conditional implements Conditional {
 
 	/**
-	 * Returns whether or not this conditional is met.
+	 * Returns whether this conditional is met.
 	 *
-	 * @return bool Whether or not the conditional is met.
+	 * @deprecated 19.12
+	 * @codeCoverageIgnore
+	 *
+	 * @return bool Whether the conditional is met.
 	 */
 	public function is_met() {
-		return \defined( 'TRIBE_EVENTS_FILE' );
+		return false;
```

Comment:
Aside from the `@deprecated 19.12` line in the function docblock, it also needs a `\_deprecated_function( __METHOD__, 'WPSEO 19.12' );` inside the `is_met()` method too